docs/source/usage/openshift.rst

```diff
@@ -0,0 +1,76 @@
+OpenShift
+=========
+
+Owners and lessees can easily install OpenShift on hardware managed through ESI through the use of the `OpenShift assisted installer`_. This guide assumes that the relevant roles have been granted access to the required Ironic actions.
```

Comment:
I think this should use title caps: "OpenShift Assisted Installer" (here and throughout the document)

Reply:
I actually thought so too, but the blog post doesn't so I was confused. I like caps better though, so I'll change it.